bin/hdfs-mesos

```diff
@@ -1,5 +1,39 @@
 #!/bin/bash
 
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+export MESOS_HDFS_DATA_DIR=${MESOS_HDFS_DATA_DIR:-"/tmp/hdfs/data"}
+export MESOS_HDFS_SECONDARY_DATA_DIR=${MESOS_HDFS_SECONDARY_DATA_DIR:-"/var/run/hadoop-hdfs"}
+export MESOS_HDFS_NATIVE_HADOOP_BINARIES=${MESOS_HDFS_NATIVE_HADOOP_BINARIES:-false}
+export MESOS_HDFS_FRAMEWORK_MNT_PATH=${MESOS_HDFS_FRAMEWORK_MNT_PATH:-"/opt/mesosphere"}
+export MESOS_HDFS_STATE_ZK=${MESOS_HDFS_STATE_ZK:-"localhost:2181"}
+export MESOS_MASTER_URI=${MESOS_MASTER_URI:-"zk://localhost:2181/mesos"}
+export MESOS_HDFS_ZKFC_HA_ZOOKEEPER_QUORUM=${MESOS_HDFS_ZKFC_HA_ZOOKEEPER_QUORUM:-"localhost:2181"}
```

Comment:
These three definitely need to be overridden for a multi-node cluster. Let's make sure that's well documented. (Thanks if it already is!)